makeflow/src/makeflow.c

```diff
@@ -1417,6 +1461,10 @@ int main(int argc, char *argv[])
 		free(log_format);
 	}
 
+	if(enforcer) {
+		makeflow_wrapper_enforcer_init(enforcer, parrot_path);
```

Comment:
I see that we have a default value for parrot_path, but do we want to force the user to set the value? If not what is the expected behavior if they don't set a value and its not in the local dir?